tools/cldr-apps/js/src/esm/cldrForumPanel.mjs

```diff
@@ -12,6 +12,11 @@ import * as cldrSurvey from "./cldrSurvey.mjs";
 import * as cldrTable from "./cldrTable.mjs";
 import * as cldrText from "./cldrText.mjs";
 
+/**
+ * Encapsulate this class name -- caution: it's used literally in surveytool.css
+ */
+const FORUM_DIV_CLASS = "forumDiv";
```

Comment:
FORUM_DIV_CLASS is moved from cldrForum to cldrForumPanel since no longer needs export